src/interactive/Tabs.jsx

```diff
@@ -7,13 +7,21 @@ import cx from 'classnames';
  */
 export class TabsTab extends React.Component {
 	render() {
-		const { children, isSelected, className, ...other } = this.props;
+		const {
+			children,
+			isSelected,
+			className,
+			isLeftSelectorLayout,
+			...other
+		} = this.props;
+		const selectedClass = isLeftSelectorLayout
```

Comment:
no need to do this if else like this its the gift of what cx profiles. Instead i would do...
```javascript
{
					'tabs-tab--selectedLeft': isSelected && isLeftSelectorLayout,
					'tabs-tab--notSelectedLeft': !isSelected && isLeftSelectorLayout,
					'tabs-tab--selected': isSelected && !isLeftSelectorLayout
}
```

Reply:
And actually you can move that object into the `cx` init below :)